resources/lta_cmd.py

```diff
@@ -0,0 +1,205 @@
+"""
+Command line utility for Long Term Archive.
+
+Run with `python -m resources.lta_cmd $@`.
+"""
+
+import argparse
+import asyncio
+import json
+import sys
+from typing import Dict
+
+from rest_tools.client import RestClient  # type: ignore
+
+from lta.config import from_environment
+
+EXPECTED_CONFIG = {
+    'LTA_REST_TOKEN': None,
+    'LTA_REST_URL': None
+}
+
+
+async def display_config(args) -> int:
+    if args.json:
+        print_dict_as_pretty_json(args.config)
+    else:
+        for key in args.config:
+            print(f"{key}:\t\t{args.config[key]}")
+    return 0
+
+
+def print_dict_as_pretty_json(d: Dict) -> None:
+    print(json.dumps(d, indent=4, sort_keys=True))
```

Comment:
I'm unsure we need json output from the lta cmd.  What might be more useful is to make each of these functions callable (as part of a public python api).

Also, having simple functions as examples of interacting with the API show others how to do the same (so if someone wanted to write this in a different language, it should be easy).

Reply:
I don't mind adding a Python API ala some `LongTermArchiveCommand` class or something, but I think this should probably be a separate card in JIRA.

As for the JSON output, I think it's useful for robots or other scripts coming at LTA from the command line. I can strip it out if we're really opposed to it, but my preference is to keep it.